package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@edx/frontend-app-course-authoring",
+  "name": "@edx/frontend-app-authoring",
   "version": "0.1.0",
   "description": "Frontend application template",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/openedx/frontend-app-course-authoring.git"
+    "url": "git+https://github.com/openedx/frontend-app-authoring.git"
```

Comment:
Confirming that this change will require an update from users if they fetch the package via the git package.

Reply:
Is that a blocker? What update is required in that case? Sorry, I'm not quite familiar with what you're referring to here.

Reply:
The case is that we don't publish to NPM so in build pipelines the package can be fetched via `git@github.com/{repo_name}`. Will updating the URL here force an update to the url that is being used in a build pipeline?

Reply:
I don't think so, because GitHub is still serving the same repo at the old URL as well as the new one. And I don't know of any tooling that reads this particular line of package.json anyways; I think it's just for displaying a link to GitHub on the NPM website if/when it gets published to NPM.